alyciacan/rancid-tomatillos
# Feature/navbar fn

#### What's this PR do?
* Add getGenres function so that navbar populates with genres pulled from the api data
#### Where should the reviewer start?
* App and Navbar components
#### How should this be manually tested?
* open in browser, watch as the navbar beautifully populates (may need to be refreshed, see below)
#### Any background context you want to provide?

#### Are there any issues?
* yes - currently there's something wrong with the async fetch call to get the genre data. 

#### Next steps:
* make the navbar <li> elements into links that work, filtering the movies
* finish up error handling for api calls

#### Questions:
